lib/resolvers/embedded.resolver.ts

```diff
@@ -87,6 +87,40 @@ export function createResolverForEmbedded(definition: Definition, field: string)
       const parent = await this.model.findById(parentId).select(field);
       return parent[field].map((item: any) => appendIdAndTransform(embeddedDefinition, item)) as any;
     }
+
+    @Mutation(() => [Typer.getObjectType(embeddedDefinition)])
+    async [`update${util.toPascalCase(definition.name)}${util.toPascalCase(field)}`](
+      @Args(
+        'input',
```

Comment:
use inputs would make more sense
```suggestion
        'inputs',
```